synapse/static/client/login/js/login.js

```diff
@@ -128,7 +128,7 @@ matrixLogin.password_login = function() {
 
 matrixLogin.onLogin = function(response) {
     // clobber this function
-    console.log("onLogin - This function should be replaced to proceed.");
+    console.warn("onLogin - This function should be replaced to proceed.");
     console.log(response);
```

Comment:
Can we also not make it log the response? We probably wouldn't want someone using a misconfigured client to e.g. post a screenshot of the warnings not noticing it's logging an access token to their account.

Reply:
Ah, just realised that!